# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/renderer/driver_utils.h
# Commit: 754436d8b3a2
# Full Hash: 754436d8b3a2023b90eb247b3a70335e74f2fdf6
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-05-09 09:14:23
# Regressor Bug: 1633628
# File Overlap Count: 3
# Description:
#   Bug 1633628 - Vender: Don't use ClearView if we previously used dual source blending on Intel gen6. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D74137
# ==============================================================================

diff -r f8fd6f0a5102 -r 754436d8b3a2 gfx/angle/checkout/src/libANGLE/renderer/driver_utils.h
--- a/gfx/angle/checkout/src/libANGLE/renderer/driver_utils.h	Fri May 08 18:39:28 2020 +0000
+++ b/gfx/angle/checkout/src/libANGLE/renderer/driver_utils.h	Fri May 08 19:28:53 2020 +0000
@@ -94,6 +94,7 @@
     uint16_t mVersionPart;
 };
 
+bool IsSandyBridge(uint32_t DeviceId);
 bool IsIvyBridge(uint32_t DeviceId);
 bool IsHaswell(uint32_t DeviceId);
 bool IsBroadwell(uint32_t DeviceId);